src/libm-tester/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+link_directories(${CMAKE_BINARY_DIR}/lib)
+include_directories(${CMAKE_BINARY_DIR}/include)
+
+# This is why a conversation about headers is needed.
+# TODO: Find way to avoid relative includes.
+include_directories(../libm)
```

Comment:
That would assume that SLEEF is the root project. In other words, you can't integrate this Cmake project with other Cmake projects (such as LLVM).